apps/spruce/src/pages/preferences/preferencesTabs/uiSettingsTab/PreferenceToggles.tsx

```diff
@@ -60,45 +61,50 @@ export const PreferenceToggles: React.FC = () => {
     window.location.reload();
   };
 
+  const handleToggleTaskReview = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const nextState = !e.target.checked;
+    sendEvent({
+      name: "Toggled task review",
+      value: nextState ? "Enabled" : "Disabled",
+    });
+    Cookies.set(DISABLE_TASK_REVIEW, nextState.toString());
```

Comment:
I think this function would benefit from launching a success toast

Reply:
Ended up needing to have a page refresh to get consistent behavior with the cookies so I think a toast wouldn't really work now